libs/dao-data/src/Profile.ts

```diff
@@ -66,3 +77,59 @@ export default class Profile {
     return await provider.lookupAddress(address);
   }
 }
+
+const profilModel = {
+  definitions: {
```

Comment:
This is a pretty massive blob of data. Would it make sense to refactor to increase readability?